dvc/command/experiments/show.py

```diff
@@ -440,9 +441,11 @@ def show_experiments(
             td.drop(col)
 
     row_styles = lmap(baseline_styler, td.column("typ"))
-    for n, row_style in enumerate(row_styles):
-        if n % 2 != 0:
-            row_style["style"] += " on grey23"
+    if theme:
+        for n, row_style in enumerate(row_styles):
+            if n % 2 != 0:
+                color = "grey85" if theme == "light" else "grey23"
+                row_style["style"] += f" on {color}"
```

Comment:
```suggestion
            if n % 2 != 0:
                color = "white" if theme == "light" else "black"
            else:
                color = "bright_white" if theme == "light" else "bright_black"
            row_style["style"] += f" on {color}"
```

We should use the standard ansi colors for white/brwhite (light grey) or black/brblack (dark grey) rather than explicit 256 color values so that users with terminal themes that override the standard colors (like solarized) get their preferred background colors.

Also we need to explicitly specify both sets of alternating backgrounds in this case, since we don't know whether the user's default background color is the bright or regular color. If we only specify every other row and picked the "wrong" bright/regular color, it would not actually alternate when the user views the table.

Reply:
I understand the reasoning. However, using 16 colors (i.e. `bright_white`) doesn't look really good, IMO, in the themes I tested. 

For example, solarized `white/bright_white`:

<img width="864" alt="Captura de pantalla 2021-12-28 a las 11 21 39" src="https://user-images.githubusercontent.com/12677733/147556530-02ecc668-c742-4931-985a-cef2ddcecbc7.png">

v.s. `grey23` on every other:

<img width="862" alt="Captura de pantalla 2021-12-28 a las 11 22 01" src="https://user-images.githubusercontent.com/12677733/147556570-8513af3c-9b9c-454c-aae8-45cccfe14076.png">

Reply:
Not sure if it's worth the complexity of adding a separate option.

Perhaps the overall alternating background is just too complicated to get it right for every scenario and we should just leave the default.

Reply:
The difference between the two bg colors in solarized isn't supposed to be too contrasty and is expected.

But FWIW In my terminal app's (iterm2 on macos) solarized light/dark and default black/white terminals, using `black/bright_black` and `white/bright_white` I still get a more distinctive alternating pattern than your example:

<img width="652" alt="Screen Shot 2021-12-28 at 7 40 47 PM" src="https://user-images.githubusercontent.com/651988/147558442-a9f6263a-430b-492e-9294-8808ca10e29b.png">
<img width="543" alt="Screen Shot 2021-12-28 at 7 41 09 PM" src="https://user-images.githubusercontent.com/651988/147558446-f0f81b78-c2d7-4c8b-bc70-ddefed3e866d.png">
<img width="542" alt="Screen Shot 2021-12-28 at 7 41 36 PM" src="https://user-images.githubusercontent.com/651988/147558458-7e1e9259-e98d-4f48-a99b-ba591fb4c090.png">
<img width="548" alt="Screen Shot 2021-12-28 at 7 42 29 PM" src="https://user-images.githubusercontent.com/651988/147558555-00ef01a6-e4ef-4f1b-a9a9-e262bfbbdf4c.png">

but yeah, the point is that doing this as a generic solution (as opposed to having proper "theme" support where users are explicitly setting all of the color values for DVC to use everywhere) is going to be difficult, and I'd be in favor of just dropping the alternating row thing entirely.

Reply:
Agreed. I'm fine with either keeping optional "themes" using ansi color codes or not implementing row colors at all.

Reply:
It looks like this theme complexity is not worth it at all, and we shouldn't introduce it at all right now. Even given that we will want this in the future, it should probably behave like normal CLI utils do and create something like DVC_COLORS or something like that (e.g. see GREP_COLORS/LS_COLORS/etc). The current implementation complicates local exp show code and will be really hard to maintain it.